docs/spelling_wordlist.txt

```diff
@@ -33,6 +33,7 @@ deserialized
 deserializers
 deserializes
 deserializing
+e
```

Comment:
Yes, it was from "i.e", but it was failing on travis and on my local (ubuntu/mint). I can revert those.